src/buildHTML.js

```diff
@@ -1491,6 +1491,22 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.transform = function(group, options) {
+    const span = groupTypes.text(group, options);
+    if (group.value.dy) {
+        const dy = group.value.dy.value;
+        span.style.top = "" + (-dy.number) + dy.unit;
+        // Not working yet:
+        if (dy.number > 0) {
+            span.height += calculateSize(dy.number, options.style);
+        } else {
+            span.depth -= calculateSize(dy.number, options.style);
+        }
```

Comment:
The idea is that the height increases if the box is moved up, and the depth increases if the box is moved down.  At least, my mental model is that height is the extent above the baseline (y-max) while depth is the extent below the baseline (y-min) -- please correct me if I'm wrong.  However, this code doesn't seem to be working, and I'm not really sure how `height` and `depth` end up getting used (or not).